index.html

```diff
@@ -169,7 +169,7 @@ <h3>CONOPY <span>&bull; Back End Dev &bull; 2015</span></h3>
     <a href="#" class="about-me-link"
       ><section class="article">
         <artcle class="about-me-item">
-          <div class="article-content">
+          <div class="article-content about-me-content">
```

Comment:
- [ ] Good job with the about section 👍 However, please make sure to add the `LET'S CONNECT` and the `social-icons list` and remove the following mentioned in the images below as given in the design guideline 🤓🔨
- [ ] Please consider using `border-radius: 8px` for contact form button as given in Figma template 🎯✨
- [ ] Also, please make sure to add the `hover` and `active` button interactions for the contact form button

### **_Desktop version_**
Your design | Design guidelines
---|  ---|
![image](https://github.com/WilfriedM01/portfolio/assets/106119105/e8b978a7-fac2-4388-8623-61058ea5c4d6) | ![image](https://github.com/WilfriedM01/portfolio/assets/106119105/8e1ae7eb-b310-45c1-801f-6585c483d41b)



### **_Mobile version_**

Your design | Design guidelines
---|  ---|
![image](https://github.com/WilfriedM01/portfolio/assets/106119105/949cd004-4305-45b5-92b6-33cc2a5bd8c8) | ![image](https://github.com/WilfriedM01/portfolio/assets/106119105/9f20db82-7b09-4ff6-886d-41b22085325e)